Tests/AuthFoundationTests/MockResponses/token-mfa_attestation.json

```diff
@@ -0,0 +1,7 @@
+{
+    "token_type": "Bearer",
```

Comment:
nit: Does `Bearer` make sense in this context? Might be good feedback for the backend team